components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -1080,7 +1112,7 @@ private Condition createConditionForSingleAttributeFilter(String domainName, Nod
      * @throws NotImplementedException Not supported filter operation
      * @throws UserStoreException
      */
-    private String[] filterUsersUsingLegacyAPIs(ExpressionNode node, int limit, int offset, String domainName)
+    private String[] filterUsersUsingLegacyAPIs(ExpressionNode node, Integer limit, int offset, String domainName)
```

Comment:
Is it necessary to change the signature

Reply:
fixed: 8f8c20292c4fac76778a8aa95490675023d7d063